README.md

```diff
@@ -1,23 +1,43 @@
+An Ribbon UI for FreeCAD, based on the PyQtRibbon library (https://github.com/haiiliin/pyqtribbon).  
+This ribbon is based on the work of HakanSeven (https://github.com/HakanSeven12/Modern-UI) for the Modern-UI workbench.  
+Current developers are:
+* Paul Ebbers (https://github.com/APEbbers/FreeCAD-Ribbon)
+* Geolta (https://github.com/geolta/FreeCAD-Ribbon)
 
-![Screenshots of FreeCAD with the Ribbon UI](Screenshot.png)
+![](https://github.com/APEbbers/FreeCAD-Ribbon/blob/main/Resources/Images/Screenshot.png)
+
+The FreeCAD ribbon provides the following functions and features:
+* Replace the default toolbars with a ribbon based on the original toolbars
+* The ribbon design is stored in a Json file for easy modification of the ribbon design.
+* A Ribbon Design dialog is provided for easy customization. (The changes are stored in the Json file). With this dialog you can:
+  * Include/exclude toolbars to be used as a panel in the ribbon
+  * Include/exclude workbenches
+  * Create your own panels based on one or more toolbars
+  * Change the order of the panels
+  * Change the order of the buttons
+  * Customize button text
+  * Set the button size to small, medium or large
+* You can apply your own stylesheet
+* Scripts are provided to help setup your own customized Ribbon
+* Much more...
+
+See the wiki page for more details and a manual on how to customize the Ribbon to your liking.
 
-This is not finished or intended for production use, but rather to look at where the limitations of this approach, implementing it as an external Python addon, lie.
 
 ## Installation
 There are two options for installing this addon:
 ### Custom Repository for Addon Manager
-Go to `Edit/Preferences/Addon-Manager` and add the custom repository `https://github.com/geolta/FreeCAD-Ribbon.git` with the branch `main`. Now you can go to the Addon Manager (`Tools/Addon-Manager`) and install "FreeCAD Ribbon" (maybe you have to update your local cache first). Now restart FreeCAD and you will see a ribbon interface :)
+Go to `Edit/Preferences/Addon-Manager` and add the custom repository `https://github.com/APEbbers/FreeCAD-Ribbon.git` with the branch `main`. Now you can go to the Addon Manager (`Tools/Addon-Manager`) and install "FreeCAD Ribbon" (maybe you have to update your local cache first). Now restart FreeCAD and you will see a ribbon interface :)
 ### Manual Installation
 Download this repository, extract the folder and copy it to the `Mod` folder of FreeCAD, detailed information can be found at the [FreeCAD Wiki](https://wiki.freecad.org/Installing_more_workbenches). Now restart FreeCAD and you will see a ribbon interface :)
 
 ## Uninstallation
 1. Remove the folder of this in the `Mod` folder of your FreeCAD installation
 1. Restart FreeCAD.
 1. When you restarted you don't see any toolbar.
-1. Create a macro.
+1. Download the file
```

Comment:
download which file?

Reply:
Changed the text to: "Create a new macro (Macro menu -> macros...-> create macro)"